benchmark.sh

```diff
@@ -7,14 +9,16 @@ if [[ $UID != 0 ]]; then
     exit 1
 fi
 
-echo "running Benchmark ans saving results to results.txt"
+echo "running Benchmark and saving results to results.txt"
+
+# prepare instance and install deps
+apt-get update
+apt-get install -y build-essential
```

Comment:
the assignment says we should use 'amazon linux ami'. i don't know which package manager it uses.